app/components/chart/footer.js

```diff
@@ -74,28 +77,41 @@ var TidelineFooter = React.createClass({
             onChange={this.props.onClickLines} /> Lines
         </label>
       </div>
-      );
-    
+    );
 
-    
-    var rightSide = this.props.chartType === 'weekly' ? showValues :
-      this.props.chartType === 'modal' ? modalOpts : null;
-    
+    var rightSide = null;
+    var bgDataToggle = null;
+
+    if (this.props.chartType === 'weekly') {
+      rightSide = showValues;
+    } else if (this.props.chartType === 'modal') {
+      if (this.props.showingSmbg) {
+        rightSide = modalOpts;
+      }
+      bgDataToggle = (
+        <TwoOptionToggle
+          left={{ label: 'BGM', state: this.props.showingSmbg }}
+          right={{ label: 'CGM', state: this.props.showingCbg }}
```

Comment:
@krystophv and I discussed today, and I explained that this is a relic of the fact that at the beginning of the "pancakes" process we weren't certain whether we might give users the option to layer both BGM and CGM data into the view together (rather than toggle between one or the other). I don't believe we've definitively decided _against_ layering, so @krystophv and I agreed we'll leave this for now. If we do definitely decide against layering in the future, we should consolidate into one variable for cleaner, safer, more understandable code. I will add a comment here in the meantime to explain why it is the way it is.